challenges/arrayMap/array-map.test.js

```diff
@@ -5,3 +5,9 @@ describe('test', () => {
     console.log(mapOne);
   });
 });
+
+describe('it will map through', () => {
+  it('maps through an array', () => {
+    expect([{}]).toEqual([{}]);
```

Comment:
I don't see that you're calling your function in this test to test it. Make sure you're actually importing the function and are invoking somewhere in the test.